commons/src/main/java/org/eclipse/kapua/commons/event/ServiceInspector.java

```diff
@@ -108,121 +108,36 @@ private ServiceInspector() {}
 
         return configurations;
     }
-//
```

Comment:
I dont' recall why we commented this code but, looking at it, I would leave it here for a while, waiting for a proper analysis about its role

Reply:
That code is commented since 1620737fdba7477313b07327106d38c3060a9725 (committed on 22/12/2017), which is the first commit containing this file. This means that this code has always been a comment in its entire life.